src/Features/Core/Portable/Debugging/DebuggingWorkspaceServiceFactory.cs

```diff
@@ -10,15 +10,12 @@ namespace Microsoft.CodeAnalysis.Debugging
     [ExportWorkspaceServiceFactory(typeof(IDebuggingWorkspaceService), ServiceLayer.Host), Shared]
     internal sealed class DebuggingWorkspaceServiceFactory : IWorkspaceServiceFactory
     {
-        private readonly IEditAndContinueService _editAndContinueServiceOpt;
-
         [ImportingConstructor]
-        public DebuggingWorkspaceServiceFactory([Import(AllowDefault = true)]IEditAndContinueService editAndContinueServiceOpt)
+        public DebuggingWorkspaceServiceFactory()
```

Comment:
>DebuggingWorkspaceServiceFactory() [](start = 15, length = 34)

Do we still need the ctor?

Reply:
Removed.

Reply:
Yes, we want importing constructors to be explicit.

---
In reply to: [285775625](https://github.com/dotnet/roslyn/pull/34027#discussion_r285775625) [](ancestors = 285775625)